code-challenges.test.js

```diff
@@ -27,26 +45,74 @@ var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saf
 
 // b) Create the function that makes the test pass.
 
+const shuffleMe = (arr) =>{
+    let shiftedArr = arr.slice(1)
+    for( let i = shiftedArr.length-1; i>0; i--){
+        let j = Math.floor( Math.random()* (i+1));
+        [shiftedArr[i],shiftedArr[j]] = [shiftedArr[j], shiftedArr[i]];
+    }
+    return shiftedArr
+}
```

Comment:
This is good work! There's a lot going on here though. Something like this you'll want to leave notes for your future self and for your PR.